torch/csrc/jit/import.cpp

```diff
@@ -238,9 +238,10 @@ TypePtr ModuleDecoder::buildType(const onnx::TypeProto& type_proto) {
   } else if (kind == "StringType") {
     return StringType::get();
   } else if (kind.find("OptionalType") == 0) {
```

Comment:
We can get rid of the ‘find()’ here now right?

Reply:
Hmmm I think maybe not, `value_type_map_` is containing all the subtype value infos, just getting the `subkind.dim_param` might not give us information on the subtype since it is just a name like `OptionalType#1` (also subtype here might be ListType or TupleType), thus need to find in the map here to get the details. In addition, this make `OptionalType` consistent with other container types like `ListType` and `TupleType`

Reply:
But the denotation itself should just be `OptionalType` so this could instead be `kind == "OptionalType"` to match the others

Reply:
Aha...sorry I thought you were talking about this statement `auto it = value_type_map_.find(subkind.dim_param());` lol agree with you on that